### ISSUE TITLE
Only the last issue in the reports files

### ISSUE THREAD

**Issue by chatelgu** [jeremymailen/kotlinter-gradle#91]
In 1.23.0, only the last issue for each files is reported inside the report file.
Functional unit test  build/reports/ktlint/main-lint.xml contains: 
```
<?xml version="1.0" encoding="utf-8"?>
<checkstyle version="8.0">
	<file name="src/main/kotlin/KotlinClass.kt">
		<error line="3" column="16" severity="error" message="Unexpected spacing before &quot;(&quot;" source="paren-spacing" />
	</file>
</checkstyle>
```

Same issue with plain and json reports
Version 1.22.0 is clean

Looks like the SortedThreadSafeReporterWrapper delegates only the last LintError added.

**Comment by jeremymailen** [comment]
Thank you for the report! I will look into it.

**Comment by jeremymailen** [comment]
Fixed in 1.23.1. Thanks again!

**Comment by chatelgu** [comment]
Thanks for the quick fix !
